### ISSUE TITLE
Explain on one-sided rules throws error

### ISSUE THREAD

**Issue by dillonjason** [northwesternmutual/regent#14]
```
const rule = {
  left: '@test',
  fn: 'empty'
}

const data = {
  test: 'thing'
}

console.log(explain(rule, data)) // throws regent.explain must be called with a regent rule​​
console.log(evaluate(rule, data)) // executes fine
```

**Comment by eisenivan** [comment]
Working on this now...

**Comment by eisenivan** [comment]
Fixed in v2.0.3
